.github/workflows/ci.yml

```diff
@@ -33,3 +33,10 @@ jobs:
         run: make -C protos lint
       - name: Test
         run: go test -race -count=1 -timeout 1h -failfast -p 1 -tags integration ./...
+      - name: Bump version and push tag
+        id: bump_version
+        uses: anothrNick/github-tag-action@1.39.0
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Do we want to set more granular [permissions](https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs) for what the GITHUB_TOKEN can be used for, or do we consider it pretty low risk at this point?